certified-connectors/Plumsail HelpDesk/apiDefinition.swagger.json

```diff
@@ -1,1884 +1,1896 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v4",
-    "title": "Plumsail HelpDesk",
-    "description": "It allows you to interact with Plumsail HelpDesk ticketing system. Visit https://plumsail.com/docs/help-desk-o365/v1.x/API/ms-flow.html for more information.",
     "contact": {
+      "email": "support@plumsail.com",
       "name": "Plumsail",
-      "url": "https://plumsail.com/sharepoint-helpdesk",
-      "email": "support@plumsail.com"
-    }
+      "url": "https://plumsail.com/sharepoint-helpdesk"
+    },
+    "description": "The API allows interact with Plumsail HelpDesk ticketing system. Visit https://plumsail.com/docs/help-desk-o365/v1.x/API/ms-flow.html for more information.",
+    "title": "Plumsail HelpDesk",
+    "version": "v4"
   },
   "x-ms-connector-metadata": [
-    {
-      "propertyName": "Website",
-      "propertyValue": "https://plumsail.com"
-    },
-    {
-      "propertyName": "Privacy policy",
-      "propertyValue": "https://plumsail.com/privacy-policy/"
-    },
-    {
-      "propertyName": "Categories",
-      "propertyValue": "Collaboration;Business Management"
-    }
+        {
```

Comment:
@RFlipper The properties in this section are not aligned properly. Please refer [this](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/365Training/apiDefinition.swagger.json#L13) swagger for correcting indentation issues in your swagger.